docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,146 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+PyDocStyleBearTest
+
+.. code::
+
+    from bears.python.PyDocStyleBear import PyDocStyleBear
+    from tests.LocalBearTestHelper import verify_local_bear
+
+
+    good_file = '''
+    """example module level docstring."""
+
+
+    def hello():
+        """Print hello world."""
+        print("hello world")
+
+    '''.splitlines(keepends=True)
+
+
+    bad_file = '''
+    """
+    example module level docstring
+    """
+
+    def hello():
+        print("hello world")
+
+    '''.splitlines(keepends=True)
+
+    PyDocStyleBearTest = verify_local_bear(PyDocStyleBear,
+                                           valid_files=(good_file,),
+                                           invalid_files=(bad_file,),
+                                           tempfile_kwargs={"suffix": ".py"})
+
+``good_file`` is a file which your bear considers as non-style-violating
+and a ``bad_file`` is one which has atleast one error/warning/info.
+``PyDocStyleBear`` checks the python docstrings. So you need to write a "good
```

Comment:
earlier(line 52) you used two double back-ticks, now you used `"` ..use only one of them everywhere in the doc, prefferably the back-ticks for good code